packages/algob/src/lib/status.ts

```diff
@@ -19,3 +19,35 @@ export async function printAssets (deployer: AlgobDeployer, account: string): Pr
   console.log("Asset Holding Info:", accountInfo.assets);
   console.log("Account's ALGO (microalgos):", accountInfo["amount-without-pending-rewards"]);
 }
+
+// read local state of application from user account (stateful smart contract)
+export async function readLocalState (
```

Comment:
This function reads and prints the local state from user account. So, should we change this function name to `printLocalStateSSC` (similar to `printAssets`, `printAssetHolding` etc) ? Same goes for `readGlobalState`.

Reply:
yes